test_tf2/test/buffer_core_test.cpp

```diff
@@ -164,7 +162,7 @@ void push_back_1(std::vector<std::string>& children, std::vector<std::string>& p
   dy.push_back(0.0);
 }
 
-void setupTree(tf2::BufferCore& mBC, const std::string& mode, const ros::Time & time, const ros::Duration& interpolation_space = ros::Duration())
+void setupTree(tf2::BufferCore& mBC, const std::string& mode, const builtin_interfaces::msg::Time & time, const tf2::TempDuration& interpolation_space = tf2::TempDuration())
```

Comment:
If you have a placeholder type for Duration, shouldn't there be one for Time as well? For example `builtin_interfaces::msg::Time` doesn't respect sim time, and if this code is meant to work with sim time at all, it will need to be replaced in the future.